templates/repo/citation_buttons.tmpl

```diff
@@ -0,0 +1,19 @@
+<button class="ui basic citation button no-transition" id="citation-copy-apa" data-text="">
+APA
+</button>
+<button class="ui basic citation button no-transition" id="citation-copy-bibtex" data-text="">
+BibTeX
+</button>
+<!-- the value will be updated by initCitationFileCopyContent, the code below is used to avoid UI flicking  -->
+<input id="citation-copy-content" value="" size="1" readonly>
+<script>
+	(() => {
+		const format = localStorage.getItem('citation-copy-format') || 'apa';
+		const btn = document.getElementById(`citation-copy-${format}`);
+		// it's ok if we don't find the btn here, initCitationFileCopyContent will take care of it
+		document.getElementById('citation-copy-content').value = btn ? btn.getAttribute('data-text') : '';
+	})();
+</script>
```

Comment:
Please try to avoid using inline script when it's not required. Please move this to `web_src/js/features/repo-common.js`. Avoiding UI flickering is not a good justification.